test/python/test_status_api.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+WHOAMI_FIELDS = [
+    "client_ip",
+    "user_agent",
+    "account",
+    "username",
+    "token_issued_at",
+]
+
+class TestWhoamiApi(api_config.ConfiguredTest):
+    """WhoamiApi unit test stubs"""
+
+    def setUp(self):
+        self.api = openapi_client.api.status_api.StatusApi(self.client)
+        self.bad_auth_api = openapi_client.api.status_api.StatusApi(self.bad_auth_client)
+
+    def test_who_am_i_200(self):
+        """Test case for who_am_i 200 response
+
+        Provides information about the client making an API request.
+        """
+        result = self.api.who_am_i()
+
+        self.assertIsInstance(result, openapi_client.models.who_am_i.WhoAmI)
+
+        for i in WHOAMI_FIELDS:
+            value = getattr(result, i)
+            # Just make sure that all the attributes have a value assigned to them
+            self.assertNotEqual(value, '')
+
+    def test_who_am_i_401(self):
+        """Test case for who_am_i 401 response
+
+        Provides information about the client making an API request.
```

Comment:
I think this comment is technically false due to the test case it's in. Might be better to describe why the test case fails in this scenario.